ActivityListener/ActivityListener.csproj

```diff
@@ -21,7 +21,7 @@
     <PackageReference Include="AWSXRayRecorder.Handlers.AwsSdk" Version="2.8.3" />
     <PackageReference Include="Hackney.Core.DynamoDb" Version="1.51.0" />
     <PackageReference Include="Hackney.Core.Logging" Version="1.49.0" />
-    <PackageReference Include="Hackney.Shared.ActivityHistory" Version="0.17.0" />
+    <PackageReference Include="Hackney.Shared.ActivityHistory" Version="0.18.0-feat-add-target-0001" />
```

Comment:
This shouldn't be feature branch please update on master then merge this PR to release